Tests/GenBank/qualifier_escaping.gb

```diff
@@ -0,0 +1,57 @@
+LOCUS       NP_034640                182 aa                     ROD 01-NOV-2000
+DEFINITION  interferon beta, fibroblast [Mus musculus].
+ACCESSION   NP_034640
+VERSION     NP_034640.1  GI:6754304
+KEYWORDS    .
+SOURCE      house mouse
+  ORGANISM  Mus musculus
+            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;
+            Mammalia; Eutheria; Rodentia; Sciurognathi; Muridae; Murinae; Mus.
+REFERENCE   1  (residues 1 to 182)
+  AUTHORS   Higashi,Y., Sokawa,Y., Watanabe,Y., Kawade,Y., Ohno,S., Takaoka,C.
+            and Taniguchi,T.
+  TITLE     structure and expression of a cloned cdna for mouse interferon-beta
+  JOURNAL   J. Biol. Chem. 258, 9522-9529 (1983)
+  MEDLINE   83265757
+COMMENT     PROVISIONAL REFSEQ: This record has not yet been subject to final
+            NCBI review. The reference sequence was derived from K00020.1.
+FEATURES             Location/Qualifiers
+     source          1..182
+                     /organism="Mus musculus"
+                     /db_xref="taxon:10090"
+                     /chromosome="4"
+                     /map="4 42.6 cM"
+                     /note="Already ""escaped"""
+     Protein         1..182
+                     /product="interferon beta, fibroblast"
+                     /note="Not ""properly"" ""escaped"""
```

Comment:
In what way is this example not properly escaped? It looks OK to me. Did you mean to have some single double-quote characters in the example?